lib/preloader/default-state.js

```diff
@@ -42,6 +42,7 @@ export default {
   schemas: {},
   locals: {},
   models: {},
+  kilns: {},
```

Comment:
Is this a typo? I mean, is it suppose to say `kilnjs` or `kilns`? 😛

Reply:
I changed it to componentKilnjs to avoid any confusion between Kiln itself and the kiln.js files that live in the components folder.